### ISSUE TITLE
'alt' property for describing image element in a herocard is not working

### ISSUE THREAD

**Issue by madbeher** [microsoft/BotFramework-WebChat#3371]
We are trying to add descriptive text to an image element in hero card using 'alt' property of 'CardImage' object as mentioned in the below document.

https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#hero-card

https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#cardimage-object

I have attached screenshot of implemented logic 
[Herocard_LG_Template.zip](https://github.com/microsoft/BotFramework-Composer/files/4992601/Herocard_LG_Template.zip)
 for the same.After implementing as shown in the attach image 'alt' property for image is not working and image is not getting displayed as well.

**Comment by v-kydela** [comment]
@tdurnford - I'm only seeing an `images` property, like I said. Can you show me where the `image` property is documented?

![image](https://user-images.githubusercontent.com/41968495/88968672-7f264b00-d264-11ea-9ce3-76617bf98b96.png)

**Comment by tdurnford** [comment]
@v-kydela Hero cards have an `images` property - not an `image` property. You are correct.

**Comment by madbeher** [comment]
We tried using both 'image' and 'images' property and both are working.But 'alt' property is not working for both of them. Below is the documentation link which mentions that herocard has 'image' property.
    
https://docs.microsoft.com/en-us/composer/how-to-send-cards

<img width="503" alt="herocard properties" src="https://user-images.githubusercontent.com/36945600/89009539-5bc5d380-d32a-11ea-89da-8dbe2b2f41ae.PNG">


Please find the attached screenshot from above documentation link which mentions that herocard has 'image' property.

**Comment by madbeher** [comment]
Please find below documentation link specific to hero card.

https://docs.microsoft.com/en-us/composer/how-to-send-cards#herocard

<img width="511" alt="herocardtemplateincomposer" src="https://user-images.githubusercontent.com/36945600/89025369-583f4600-d344-11ea-8c46-0ffdb38350a7.PNG">

**Comment by compulim** [comment]
Alt-text are something outside for `title`, `subtitle`, and `text`. And the schema doesn't support it.

We need to bring in the service team to update the schema to provide support on accessibility.

**Comment by compulim** [comment]
Few things:

1. The Composer doc is wrong, there is no `image` property, but [`images` array](https://github.com/microsoft/botframework-sdk/blob/master/specs/botframework-protocol/botframework-channel.json#L1509).
   - `images` array has 3 fields: `url`, `alt`, and `tap`
1. Although Web Chat receive `alt`, our [`AdaptiveCardBuilder.ts/addImage`](https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardBuilder.ts#L126) does not use the `altText` field. This should be fixed in Web Chat.

**Comment by zxyanliu** [comment]
FYI. The [HeroCard template](https://docs.microsoft.com/en-us/composer/how-to-send-cards#herocard) in the Composer documentation was referenced from the [sample](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/plugins/samples/assets/projects/RespondingWithCardsSample/language-generation/en-us/common.en-us.lg#L7).

**Comment by tdurnford** [comment]
I believe LG supports both the `image` and `images` properties.
